numpyro/infer/elbo.py

```diff
@@ -569,6 +574,8 @@ def single_particle_elbo(rng_key):
             model_trace, guide_trace = get_importance_trace(
                 seeded_model, seeded_guide, args, kwargs, param_map
             )
+            check_model_guide_match(model_trace, guide_trace)
+            _validate_model(model_trace, plate_warning="strict")
```

Comment:
why strict here?

Reply:
I'm seeing TraceGraphELBO leverages plate stacks etc to do inference. I'm not sure if strict is required. If you think using loose is enough, I can make the change.

Reply:
Let me merge this first, then would make the change if needed in #1310